xgi/convert.py

```diff
@@ -146,22 +147,30 @@ def convert_to_line_graph(H):
     ----------
     H : Hypergraph
         The hypergraph of interest
+    s : int
+        The intersection size to consider edges
+        as connected, by default 1.
 
     Returns
     -------
     LG : networkx.Graph
          The line graph associated to the Hypergraph
+
+    References
+    ----------
+    "Hypernetwork science via high-order hypergraph walks"
+    by Sinan G. Aksoy, Cliff Joslyn, Carlos Ortiz Marrero, Brenda Praggastis & Emilie Purvine.
+    https://doi.org/10.1140/epjds/s13688-020-00231-0
     """
 
     LG = nx.Graph()
 
     edge_label_dict = {tuple(edge): index for index, edge in H._edge.items()}
 
-    nodes = sorted(set(edge_label_dict.values()))
-    LG.add_nodes_from(nodes)
+    LG.add_nodes_from(H.edges)
```

Comment:
The nodes might be "unordered" now. Better to keep them ordered?

Also, just realised some of those nodes might not get edges and stay isolated. Not sure if we want that here or not.

Reply:
I intentionally removed the sorting because there will be cases where sorting isn't possible. (Mixes of strings and ints for example)

Reply:
Exactly. The `vector_centrality` measure is downstream of this method and relies on the line graph having the same nodes, so I will keep it as is for now.